os/kernel/environ/env_dup.c

```diff
@@ -113,17 +113,38 @@ int env_dup(FAR struct task_group_s *group)
 
 	/* Does the parent task have an environment? */
 
-	if (ptcb->group && ptcb->group->tg_envp) {
-		/* Yes..The parent task has an environment, duplicate it */
+	if (ptcb->group != NULL && ptcb->group->tg_envp != NULL) {
+		/* Yes.. The parent task has an environment allocation. */
 
 		envlen = ptcb->group->tg_envsize;
-		envp   = (FAR char *)kumm_malloc(envlen);
-		if (!envp) {
-			ret = -ENOMEM;
-		} else {
-			group->tg_envsize = envlen;
-			group->tg_envp = envp;
-			memcpy(envp, ptcb->group->tg_envp, envlen);
+		envp   = NULL;
```

Comment:
redundant. This line is necessary on Nuttx change. Not in here.

Reply:
removed